### DISCUSSION TITLE
Non-torrent file downloader?

### DISCUSSION THREAD

**Discussion by GitTheHellOutaHere** [transmission/transmission#5215] [Ideas]
Is Transmission going to remain strictly a torrent client and never become a multi-purpose downloader like Folx? Even if Transmission didn't include an extension to capture browser links, it would still be cool if it could download non-torrent files. Countless music downloads are available in rar format.

In Folx I'm able to drag or paste download links for faster downloading than in a browser. Transmission throws an error saying _"it appears the the file...is not a torrent file."_ 

Transmission downloads much faster than Folx and it would be nice if it could download the same files so I could get rid of Folx altogether.

**Comment by g-regor** [comment]
> Transmission downloads much faster than Folx and it would be nice if it could download the same files so I could get rid of Folx altogether.

With regular downloads, the bottle neck is the server's upload speed, so I doubt that the problem is Folx and that Transmission would be able to make it faster.

**Comment by GitTheHellOutaHere** [comment]
Doubt all you want, but that's the way it is.

That's like saying two different cars should go the same speed - the only bottleneck is the road. _No_, the speed limitations are primarily internal, not external.

Or two phones should download at the same speed. _NO_,  my old Android 8 phone downloads slower than a new 'droid 14 model, and probably slower than most iPhones. The bottleneck isn't the server, it's the hardware/software.

**Unless you've inspected the code of each app, you don't know what bottlenecks may exist.**

Two browsers don't work the same and those two apps don't work the same. Both were running on the same computer, same IP addresses and neither were speed restricted. Whether running individually or simultaneously, Transmission is faster. Large video downloads finish 1/3 or more faster than Folx.

I've used both for years and am quite familiar with their settings. I stopped using Transmission for a long time because Folx is an all-purpose downloader, although it seemed noticeably slower. When I began comparing it to Transmission, the speed difference was clear.

One thing I like about Folx is it shuts down faster. Transmission often takes 20 seconds or more to close even when none are actively downloading (like when my vpn drops out and there's no connection and I need to quickly close the app). Sometimes I've force quit rather than wait in case my IP address was leaking. Another is its ability to D/L rar files from download links in a browser - just copy link & paste.
